common/tools/field_test_log_plotter/ftl_plotter.py

```diff
@@ -94,6 +94,9 @@ def __create_dataframe(self):
                                                         inplace=True)
         # Drop any DF lines where timestamp is NaN
         self.df.dropna(subset=['Timestamp'], inplace=True)
+        # Drop corrupted DF lines i.e. the ones where the
+        # last expected csv file column is empty.
+        self.df.dropna(subset=['3v3 current [mA]'], inplace=True)
```

Comment:
How about using "df.columns[-1]" for last column selection?  Instead of hardcoded column name.

Reply:
That is definitely better way to do it.